templates/applications.html

```diff
@@ -57,7 +57,7 @@ <h3 class="page-header">Applications for {{ page.title }}</h3>
                   {% for app in applications %}
                   <tr>
                       <td><input type="checkbox" name="application" value="{{ app.id }}" /></td>
-                      <td><a href='{{ app.id }}'>{{ app.number }}</a></td>
+                      <td><a href='{{ app.number }}'>{{ app.number }}</a></td>
```

Comment:
I know it was this way already, but we should really be using `{% url %}` for this.